game/templates/game/lesson_detail.html

```
@@ -0,0 +1,393 @@
+            <a href="{% url 'lesson_detail' next_lesson %}">
+                {{ next_lesson }} →
+            </a>
+        {% endif %}
+    </div>
+
+</div>
+
+
+</div>
+
+</div>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)